setup.py

```diff
@@ -89,6 +89,7 @@ def read(f):
     "pytest-timeout",
     "async-generator",
     "pytest-xdist",
+    "proxy.py",
```

Comment:
Don't forget to add this to https://github.com/aio-libs/aiohttp/blob/master/requirements/test.txt.

Reply:
I'll put it in there, but should I just remove this section altogether?

Reply:
Let's not do any section removals or packaging refactoring in this PR. It may be reasonable to do this separately, though.
Ref: https://github.com/aio-libs/aiohttp/pull/3518#issuecomment-453128368.